maat-court-data-api/src/main/resources/db/changelog/changes/grant_mla_permissions_for_drc_integration_testing.sql

```diff
@@ -0,0 +1 @@
+GRANT SELECT, INSERT, UPDATE, DELETE ON togdata.FDC_ITEMS TO MLA;
```

Comment:
I'm not sure the best thing to do is grant permissions to another schema - why not just use the togdata schema?

Reply:
I thought the maat-api service used MLA for running it's queries? It seems in the broader scope of the db/changelog/changes that that is the case.
Is this not the case?